src/DeviceRenderer.js

```diff
@@ -40,7 +41,6 @@ module.exports = class DeviceRenderer {
 
         // Websocket
         this.webRTCWebsocket = null;
-        this.webRTCWebsocketName = 'gm-webrtc';
```

Comment:
It could be preferable to defined it but set it  to null

Reply:
You're right, it'll be more legible. But first I'll need to test if setting up the webrtc socket with a `null` protocol is the same as setting it up with an `undefined` one.

Reply:
@JulienBolard setting the protocol to `null` doesn't work, the socket is initialized with a `null` protocol instead of without a specific one, and so the connexion isn't established.